src/GraphQL.SystemTextJson/SchemaExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Threading.Tasks;
+using GraphQL.Types;
+
+namespace GraphQL.SystemTextJson
+{
+    public static class SchemaExtensions
+    {
+        /// <summary>
+        /// Configures an <see cref="ExecutionOptions"/> using the given <paramref name="configure"/> action
+        /// then executes those options using the <paramref name="schema"/> and a <see cref="DocumentWriter"/>
+        /// with indentation turned on. 
+        /// </summary>
+        /// <param name="schema">A schema to use.</param>
+        /// <param name="configure">An action that configures something to execute.</param>
+        /// <returns>The JSON result as a string.</returns>
+        /// <remarks>
+        /// Useful for quickly executing something and "getting started".
+        /// Part of the public API and should not be removed even if it has no references.
+        /// </remarks>
+        public static Task<string> ExecuteAsync(this ISchema schema, Action<ExecutionOptions> configure)
+            => schema.ExecuteAsync(new DocumentWriter(indent: true), configure);
```

Comment:
Since all calls to `public static async Task<string> ExecuteAsync(this ISchema schema, IDocumentWriter documentWriter, Action<ExecutionOptions> configure)` pass not null delegate (at least `Query` property must be set) consider to add 
```c#
 if (configure == null)
                throw new ArgumentNullException(nameof(configure));
```
in the beginning of ` public static async Task<string> ExecuteAsync(this ISchema schema, IDocumentWriter documentWriter, Action<ExecutionOptions> configure)`

Reply:
Yep, makes sense. Not 100% foolproof as you elude to, but better than nothing.